project.py

```diff
@@ -120,12 +126,11 @@ def _append_item_to_project(self, item):
         # retrieve jira components and labels as github labels
         labels = []
         for component in item.component:
-            labels.append('jira-component:' + component.text.lower())
-            labels.append(component.text.lower())
+            if os.getenv('JIRA_MIGRATION_INCLUDE_COMPONENT_IN_LABELS', 'true') == 'true':
+                labels.append('jira-component:' + component.text.lower())
+                labels.append(component.text.lower())
 
-        #labels.append('type:' + component.text)
-        for ltype in self._project['Types'].keys():
-            labels.append('jira-type:' + component.text)
```

Comment:
there was a typo here that it used the component rather than type

Reply:
That explains why there were no diff between "jira-component:*" and "jira:type:*" labels on imported INFRA issues... 😅